radis/io/kurucz.py

```diff
@@ -0,0 +1,162 @@
+""""
+
+Summary
+-----------------------
+
+Kurucz database parser
+
+-----------------------
+
+Defines :py:func:`~radis.io.fetch_kurucz` based on :py:class:`~AdBKurucz`
+
+"""
+
+import os
+from os.path import abspath, expanduser, join
+
+from radis.api.hdf5 import DataFileManager
+from radis.api.kuruczapi import AdBKurucz,get_atomic_number,get_ionization_state, KuruczDatabaseManager
+
+def fetch_kurucz_new(
```

Comment:
see my other comment : there should not be two functions "old" and "new". Just replace the old code with the new one.

Reply:
already fixed: https://github.com/radis/radis/pull/652/commits/addb90c0dc2fe3bdfb915defc6e548802ef0df1f#diff-8694bc131153e428bcabb11b9ff0b5ab0bc832e1b1e5f9431a5c95c936fba48eR21